File: src/test/java/com/leetcode/top/interview/medium/UniquePathsTest.java
package com.leetcode.top.interview.medium;

import com.leetcode.dp.limited.UniquePaths;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

/**
 * @author wezhyn
 * @since 07.24.2020
 */
public class UniquePathsTest {

    @Test
    public void uniquePaths() {
        assertEquals(3, new UniquePaths().uniquePaths(3, 2));
        assertEquals(28, new UniquePaths().uniquePaths(7, 3));
        assertEquals(2000000000, new UniquePaths().uniquePaths(100, 100));
    }

}

File: src/test/java/com/daily/UniquePathsTest.kt
package com.daily

import org.junit.Assert.assertEquals
import org.junit.Test

/**
 *
 * @author wezhyn
 * @since 08.16.2020
 */
class UniquePathsTest {

    @Test
    fun uniquePaths() {
        assertEquals(2000000000, UniquePaths().uniquePaths(50, 50))
        assertEquals(28, UniquePaths().uniquePaths(7, 3))
    }
}